src/wp-includes/class-wp-duotone.php

```diff
@@ -998,11 +986,12 @@ private static function get_selector( $block_name ) {
 	 *
 	 * @since 6.3.0
 	 */
-	public static function set_global_styles_presets() {
+	public static function load_global_styles_presets() {
```

Comment:
I see you're updating a public method name. Is there a way to mark these public methods as "for internal use only"?

I asked when this was being backported (https://github.com/WordPress/wordpress-develop/pull/4619#issuecomment-1611512758), but it didn't get added.

I wanted to leave room for changes like the ones in this PR without having to maintain backwards compatibility of the public methods. These are more implementation details than anything, but they have to be public to be used by hooks.

Reply:
> Is there a way to mark these public methods as "for internal use only"?

Ideally we should just actually mark it as private, everything else is bad design.

Reply:
I guess this one could be private now since it isn't being used in a hook, but other hooks are still needed and you can't call a hook with a private method.

Reply:
If these methods are not public, let's merge the methods.